0000-pr-merge-requirements.md

```diff
@@ -0,0 +1,55 @@
+- Feature Name: PR Merge Requirements
+- Start Date: 2019-05-21
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Lay out the requirements to merge a PR
+
+# Motivation
+[motivation]: #motivation
+
+As we begin our working groups, to make sure that our community is all on the same page, we want to
+clearly define our requirements to merge a PR. This should benefit developers, reviewers and those
+merging the PRs.
+
+# Design
+[design]: #detailed-design
+
+All PR requirements
+  - Approval Reviews: 1 approval review from core team member OR
+                      1 approval review from captain of working group
+  - All tests pass.
```

Comment:
Some windows tests and some pylint are bit fragile. Also the CI sometimes fail because other random reasons.

I support this requirement: we cannot merge with all the tests passing, but I wonder what can we do about the cases where CI is not ready. Is there any mechanism where the contributor can help here?

Reply:
If there is a case where we are okay with the tests not passing due to CI issues this would fall under the exception to require 3 reviews. 

> Is there any mechanism where the contributor can help here?

If there is a test that is failing that is not related to the PR you can definitely dive in and figure out why and push a separate PR to try to fix, but ultimately that responsibility lays on our team. If windows or pylint tests are failing constantly we need to dive in and figure out why. I think it makes sense to add some more documentation around how we have these tests setup so its easier for contributors to try to run them with the same infrastructure as us. I'll add the step to document this to this SEP.

Reply:
> I think it makes sense to add some more documentation around how we have these tests setup so its easier for contributors to try to run them with the same infrastructure as us. I'll add the step to document this to this SEP.

Thanks, I love that.